src/index.js

```diff
@@ -117,7 +118,7 @@ function reallyInitializeApp(routes) {
 export default function initializeApp(routes) {
   // set up logging when you're in production mode
   if (isProdMode()) {
-    Raven.config('https://e19420a2c46a4f97942553dfe8322cc4@sentry.io/1229723', {
+    Raven.config(process.env.SENTRY_DSN, {
```

Comment:
Not a blocker @VinneyJ but isn't it better if we define the sentry DSN as a const, rather than passing it here directly

Reply:
@thepsalmist sure I can do that.

Reply:
It's ready for review and merging. @kilemensi